metrics/details/init.lua

```diff
@@ -99,6 +99,21 @@ end
 
 global_metrics_registry = Registry.new()
 
+------------------------------- Global Labels --------------------------------
+global_labels = {}
+
+local function append_global_labels(label_pairs)
+    local extended_label_pairs = table.deepcopy(label_pairs)
```

Comment:
Why not simple "copy"?

Reply:
Changed both `table.deepcopy` calls to `table.copy` calls.

Reply:
We also do not call excessive copy in case of empty global labels now